projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -242,7 +242,7 @@ private static function stats_array_to_string( $kvs ) {
 		foreach ( $kvs as $k => $v ) {
 			$jskvs[] = "$k:'$v'";
 		}
-		return join( ',', $jskvs );
+		return implode( ',', $jskvs );
```

Comment:
I'm curious about this. Is `join` discouraged as of PHP 8? I'm not against that change, I'd much rather we use `implode` everywhere, but I don't understand how that's linked to PHP 8? Is `join` going to be deprecated maybe?

Reply:
I think `join` to `implode` is less a PHP 8-specific thing and more of a "standardize across the codebase" thing. I'm not sure who decided on these but there are a few similar substitutions that you can see in https://github.com/Automattic/wpcom-php-migration/blob/main/rector.php#L57

Reply:
Working on the phpcs change here: https://github.com/Automattic/jetpack/pull/30601